src-v2/Judge/NTS.Judge.Blazor/Pages/Dashboard/Startlist/UpcomingStarts.razor

```diff
@@ -0,0 +1,79 @@
+﻿@using NTS.Domain.Core.Objects
+@using NTS.Judge.Blazor.Ports
+@using System.Collections
+@using Not.Blazor.TM
+@using Not.Extensions
+
+@inherits NotComponent
+
+@if (Items.Count() == 0)
+{
+    <MudText Typo="Typo.body1">There aren't any upcoming starts at the moment.</MudText>
+}
+else
+{
+    <MudTabs @ref="@Tabs" PanelClass="px-4 py-6" Elevation="4" Rounded ApplyEffectsToContainer>
+        @foreach (var tab in UserTabs)
+        {
+            <MudTabPanel ID="@tab.Id" Text="@tab.Label">
+                <ChildContent>
+                    <MudSimpleTable Dense="@dense" Hover="@hover" Bordered="@bordered" Striped="@striped" Style="overflow-x: auto;">
+                        <thead>
+                            <tr>
+                                @foreach (var h in headings)
```

Comment:
Is this a good name?

Reply:
Extracted MudSimpleTable usage in NotSimpleTable